doc/sphinxext/numpydoc/numpydoc.py

```diff
@@ -42,13 +42,6 @@ def mangle_docstrings(app, what, name, obj, options, lines,
                class_members_toctree=app.config.numpydoc_class_members_toctree,
               )
 
-    # PANDAS HACK (to remove the list of methods/attributes for Categorical)
-    no_autosummary = [".Categorical", "CategoricalIndex", "IntervalIndex",
-                      "RangeIndex", "Int64Index", "UInt64Index",
-                      "Float64Index", "PeriodIndex", "CategoricalDtype"]
-    if what == "class" and any(name.endswith(n) for n in no_autosummary):
-        cfg['class_members_list'] = False
-
```

Comment:
I *think* there is also some other code related to this hack that then could be removed (eg the place where I define the 'class_members_list' config). See https://github.com/pandas-dev/pandas/commit/cf40991093e746e5dde40fca4f322ecfc3dfff78 that added it